network/sites.csv

```diff
@@ -1476,12 +1476,15 @@ NTVZ,31,-39.0984817,175.6759733,1260,,WGS84,Unknown,2016-01-14T02:10:00Z,9999-01
 NTVZ,32,-39.0984817,175.6759733,1260,,WGS84,Unknown,2016-01-14T02:10:00Z,9999-01-01T00:00:00Z
 NWEZ,10,-39.27488,173.86743,230,,NZGD2000,Topographic Map,1996-04-03T17:09:00Z,2012-10-16T00:00:00Z
 NWFS,20,-41.589065731,175.233958687,14,,WGS84,Internal GPS Clock,2006-05-05T02:01:00Z,9999-01-01T00:00:00Z
-NZA,40,-42.3707,176.9109,0,2690,WGS84,Unknown,2019-12-22T00:00:00Z,9999-01-01T00:00:00Z
+NZA,40,-42.3707,176.9109,0,2690,WGS84,Unknown,2019-12-22T00:00:00Z,2021-12-17T23:59:59Z
+NZA,41,-42.3717,176.9093,0,2635,WGS84,Unknown,2021-12-18T00:00:00Z,9999-01-01T00:00:00Z
 NZAS,20,-46.586807,168.378357,13,,WGS84,Internal GPS Clock,2008-10-08T22:00:00Z,2015-06-30T23:59:59Z
 NZB,40,-40.6003,179.0996,0,3184,WGS84,Unknown,2020-09-18T00:00:00Z,9999-01-01T00:00:00Z
-NZC,40,-38.2001,-179.7978,0,3545,WGS84,Unknown,2019-12-13T00:00:00Z,9999-01-01T00:00:00Z
+NZC,40,-38.2001,-179.7978,0,3545,WGS84,Unknown,2019-12-13T00:00:00Z,2021-12-10T23:59:59Z
+NZC,41,-38.2004,-179.7978,0,3545,WGS84,Unknown,2021-12-11T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
In `sites.csv`  NZC 41  lat should be `-38.204` 
(picky me:))

Reply:
Hey @salichon pretty sure in voyage logs it is `38 12.023 S` (degrees minutes) so then that is `-38.20038` decimal degrees, so rounded is `-38.2004`, how did you get `-38.204`?

Reply:
All good i miiplace  the  decimal separator ....

Reply:
No worries! Just wanted to check :)